services/machine_learning/requirements.txt

```diff
@@ -0,0 +1,5 @@
+numpy
```

Comment:
Versions of plugins should probably be specified, to ensure an update to these packages doesn't break something down the line